GoogleForCreators/web-stories-wp
# Templates: Re-sort templates so that new ones are at the top

## Context
Part of making new templates more visible to users

## Summary
- Adds `creationDate` to all templates
- Adds a smoll unit test to make sure there's a valid `creationDate` on all the templates
- Updates the template creation README to include `creationDate`
- Sorts templates by `creationDate` , newest appearing first.

## Relevant Technical Choices
NA


## To-do
- [x] go through and see if we want any more specific ordering of templates beyond "new ones at the top"


## User-facing changes
Templates should now appear in a new order where the original templates are at the bottom.


## Testing Instructions
- Open up `Explore Templates`
- Scroll to the bottom and see that the original templates are at at the bottom of the template gallery with newer ones on top


### QA
- Open up `Explore Templates`
- Scroll to the bottom and see that the original templates are at at the bottom of the template gallery with newer ones on top






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8465
